Makefile

```diff
@@ -0,0 +1,37 @@
+########################################################################################################
+
+SHELL=bash
+NAME = zimbra-help
+DESC = Package for getting help files installed for web client users.
+
+########################################################################################################
+
+all: zimbra-help-pkg
+
+########################################################################################################
+
+stage-help-files:
+	mkdir -p build/stage/$(NAME)/opt/zimbra/jetty_base/webapps/zimbra/help/
+	cp -r en_US build/stage/$(NAME)/opt/zimbra/jetty_base/webapps/zimbra/help/en_US
+	cp -r de build/stage/$(NAME)/opt/zimbra/jetty_base/webapps/zimbra/help/de
```

Comment:
nit: alphabetical order may be better.

Reply:
Updated the order. @k-kato  Can you please check ?